### PR TITLE
Some documentation for Connection Pooling in the SQL Module

### PR DESCRIPTION
**Pull request by adamretter**
Related to https://github.com/eXist-db/exist/pull/3831

### CODE REVIEW

File: src/main/xar-resources/data/extensions/extensions.xml [hunk 1]
```diff
@@ -48,286 +50,10 @@
...
-                    <para>Class: <code>org.exist.xquery.modules.spatial.SpatialModule</code> </para>
-                </listitem>
-                <listitem>
-                    <para>Namespace: <code>http://exist-db.org/xquery/spatial</code> </para>
-                </listitem>
-            </itemizedlist>
-        </sect2>
+        <title>Extension Modules Configuration</title>
+        <para>Whilst most extension modules work without any further configuration than that
+            described above, some modules have additional configuration options. Thos additional
```
Comment (line-o) [inline src/main/xar-resources/data/extensions/extensions.xml:55]: typo in word `Those`

**Comment by line-o** [comment]
I am wondering why you decided to remove the existing extension examples. Where they outdated or something?

**Comment by adamretter** [comment]
@line-o Thanks I fixed the typos

**Comment by adamretter** [comment]
> I am wondering why you decided to remove the existing extension examples. Where they outdated or something?

Yes, exactly. This was discussed in - https://github.com/eXist-db/exist/pull/3831

**Review by line-o — APPROVED** [review]

**Pull request merged**